contributions/presentation/week2/aronber-fbjorkma/README.md

```diff
@@ -0,0 +1,22 @@
+# Presentation - Monitoring
+
+## Members
+
+Aron Hansen Berggren (aronber@kth.se)
+Github username: arxra
+
+Fredrik Björkman (fbjorkma@kth.se)
+Github username: fbjorkman
+
+## Monitoring and why system feedback matters
+
+We want to present on why monitoring your pipelines and automations matter,
+as there seems to be an undervaluation of feedback.
+
+We aim to answer questions such as:
+
+- What is monitoring?
+- What are metrics?
+- What is the value of deploying often?
+- What is the value of the feedback from the system?
+- How to act upon the feedback gained from the system?
```

Comment:
The topic is too broad. Please focus on explaining this based on a real monitoring tool.